yambol/yambol/db_types.py

```
@@ -0,0 +1,93 @@
+  # Strings to add at the bottom of the file
+  bottom: list[str] = field(default_factory=list)
+  # Strings to adzd at the top of the generated file
+  top_once: list[str] = field(default_factory=list)
+  # Strings to add before creating the entity
+  before_once: list[str] = field(default_factory=list)
+  # Strings to add after creating the entity
+  after_once: list[str] = field(default_factory=list)
+  # Strings to add at the bottom of the file
+  bottom_once: list[str] = field(default_factory=list)
+  
+  def __init__(self, name, id=False, check="", inline="", fk="", index="", top=[], before=[], after=[], bottom=[], top_once=[], before_once=[], after_once=[], bottom_once=[]):
```

Ruff (0.8.2)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)